src/utils/ProfileProvider.tsx

```diff
@@ -66,40 +143,62 @@ export default function ProfileProvider({ children }: { children: ReactNode }) {
 
 /**
  * EXAMPLE USAGE:
- *
- * app/test/layout.tsx:
  */
 
-/**
- * 'use client';
- *
- * import { ReactNode } from 'react';
- * import ProfileProvider from '@/utils/ProfileProvider';
- *
- * export default function TestLayout({ children }: { children: ReactNode }) {
- *   return <ProfileProvider>{children}</ProfileProvider>;
- * }
- */
+/** app/test/layout.tsx:
 
-/**
- * app/test/page.tsx:
- */
+'use client';
 
-/** 'use client';
- *
- * import { useContext, useState } from 'react';
- * import { ProfileContext } from '@/utils/ProfileProvider';
- *
- * export default function Page() {
- *   const profile = useContext(ProfileContext);
- *   const [name, setName] = useState("");
- *
- *   return (
- *     <div>
- *       <h1>{(profile && profile.profileData) ? profile.profileData.first_name : 'NULL'}</h1>
- *       <input type="text" onChange={e => setName(e.target.value)} />
- *       <button type="button" onClick={() => profile?.updateProfile({ first_name: name })}>Change Name</button>
- *     </div>
- *   );
- * }
- */
+import { ReactNode } from 'react';
```

Comment:
Tested this out and was able to change my name in the `profiles` table! Thanks for including these.